### DISCUSSION TITLE
[Enhancement] Implement a package of .NET MAUI with Visual Code on Fedora 32.

### DISCUSSION THREAD

**Discussion by catafest** [dotnet/maui#203] [General]
## Summary
Implement a package of .NET MAUI with Visual Code on Fedora 32.

## API Changes

Just Linux API to build Linux application for Fedora 32 distro;

## Intended Use Case
I think is easy to create by the development team of this source code and will increase the area of interest for development with these source code.

**Comment by knocte** [comment]
> Visual Code 

You mean Visual Studio Code?
